src/renderer/pages/toast/modules/keyboard.js

```
@@ -0,0 +1,178 @@
+      }
+      break;
+    case '_': // Delete page in settings mode (supporting Shift+-)
+      if (isSettingsMode && event.shiftKey) {
+        event.preventDefault();
+        removePage();
+      }
+      break;
+    default:
+      // Check if key matches a button shortcut
+      // Use event.code to get physical key location instead of event.key
+      const keyCode = event.code;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)